modules/payments/public/lib/queries/transactions/search.liquid

```diff
@@ -1,25 +1,18 @@
 {% comment %}
-  SEarches for transctions.
+  Searches for transctions.
 
   Params:
     - limit Int
       default: 20
     - page: Int
       default: 1
-    - payable_ids: [String!]
-    - amount_cents: String
-    - payer_id: String
-    - currency: String
-    - gateway: String
-    - c__status: String
-    - with_gateway_requests: Boolean = false
-    - with_statuses: Boolean = false
+    - params: Hash
 {% endcomment %}
 {% liquid
-  assign page = page | to_positive_integer: 1
-  assign limit = limit | default: 20
+  assign page = page | default: params.page | to_positive_integer: 1
+  assign limit = limit | default: params.limit | default: 20
 
-  graphql r = 'modules/payments/transactions/search', limit: limit, page: page, payable_ids: payable_ids, amount_cents: amount_cents, currency: currency, gateway: gateway, with_gateway_requests: with_gateway_requests, with_statuses: with_statuses, payer_id: payer_id
+  graphql r = 'modules/payments/transactions/search', args: params, limit: limit, page: page
```

Comment:
Is it better? I liked passing just an variable to the search function without having to construct the object before.
Maybe we can have both?